### PR TITLE
refactor(docs-ui): doc link

### PR DESCRIPTION
**Pull request by weird94**
## Breaking changes
1. Split tags \x1e & \x1f will be removed from body.datastream, this was a stack design, caused too much problems on docs.
2. No breaking changes on facade api and current command.
3. Old documents can work as before.

## Pull Request Checklist

- [ ] Related tickets or issues have been linked in the PR description (or missing issue).
- [ ] [Naming convention](https://github.com/dream-num/univer/blob/dev/docs/NAMING_CONVENTION.md) is followed (**do please** check it especially when you created new plugins, commands and resources).
- [ ] Unit tests have been added for the changes (if applicable).
- [ ] Breaking changes have been documented (or no breaking changes introduced in this PR).

### CODE REVIEW

File: packages/docs-ui/src/commands/commands/clipboard.inner.command.ts [hunk 1]
```diff
@@ -101,7 +104,7 @@ export const InnerPasteCommand: ICommand<IInnerPasteCommandParams> = {
         if (docDataModel == null || originBody == null) {
             return false;
         }
-        const selections = originSelections.map((range) => BuildTextUtils.selection.getInsertSelection(range, originBody));
+        const selections = originSelections.map((range) => range);
```
Comment (wzhudev) [inline packages/docs-ui/src/commands/commands/clipboard.inner.command.ts:107]: What's the purpose of this? If you would like to copy this array, there are more explicit ways.

File: packages/core/src/docs/data-model/text-x/build-utils/text-x-utils.ts [hunk 2]
```diff
@@ -351,18 +244,47 @@ export const replaceSelectionTextX = (params: IReplaceSelectionTextXParams) => {
     const body = doc.getSelfOrHeaderFooterModel(segmentId)?.getBody();
     if (!body) return false;
 
+    const oldBody = selection.collapsed ? null : getBodySlice(body, selection.startOffset, selection.endOffset);
+    const diffs = textDiff(oldBody ? oldBody.dataStream : '', insertBody.dataStream);
```
Comment (wzhudev) [inline packages/core/src/docs/data-model/text-x/build-utils/text-x-utils.ts:248]: This util function has not tests covered.

File: path/to/file [hunk 3]
```diff
@@ -0,0 +1 @@
+
```
Comment (wzhudev) [inline path/to/file:1]: Delete this file.

File: packages/core/src/docs/data-model/text-x/build-utils/parse.ts [hunk 4]
```diff
@@ -28,8 +28,6 @@ const tags = [
     DataStreamTreeTokenType.TABLE_CELL_END, // 表格开始
```
Comment (wzhudev) [inline packages/core/src/docs/data-model/text-x/build-utils/parse.ts:28]: Are some token types deprecated? If so, you should decorate `@deprecated` on them.

**Review by wzhudev — CHANGES_REQUESTED** [review]
There should be a comprehensive document telling how the data stream is constructed.

File: packages/core/src/docs/data-model/text-x/build-utils/text-x-utils.ts (hunk 2 above)
Comment (wzhudev) [reply packages/core/src/docs/data-model/text-x/build-utils/text-x-utils.ts:248]: I mean the whole replace function.

Comment (weird94) [reply packages/core/src/docs/data-model/text-x/build-utils/text-x-utils.ts:248]: added

File: packages/core/src/docs/data-model/text-x/__tests__/transform.spec.ts [hunk 5]
```diff
@@ -724,7 +724,7 @@ describe('transform()', () => {
                     dataStream: '',
                     customRanges: [
                         {
-                            startIndex: -1,
+                            startIndex: 0,
                             endIndex: 0,
```
Comment (Jocs) [inline packages/core/src/docs/data-model/text-x/__tests__/transform.spec.ts:728]: 为啥会有 startIndex === endIndex ？

File: packages/core/src/docs/data-model/text-x/__tests__/transform.spec.ts [hunk 6]
```diff
@@ -746,7 +746,7 @@ describe('transform()', () => {
                     dataStream: '',
                     customRanges: [
                         {
-                            startIndex: -1,
+                            startIndex: 0,
```
Comment (Jocs) [inline packages/core/src/docs/data-model/text-x/__tests__/transform.spec.ts:749]: 同上，为啥 startIndex === endIndex 情况？

File: packages/core/src/docs/data-model/text-x/__tests__/transform.spec.ts (hunk 5 above)
Comment (weird94) [reply packages/core/src/docs/data-model/text-x/__tests__/transform.spec.ts:728]: 长度为1

File: packages/core/src/docs/data-model/text-x/__tests__/transform.spec.ts (hunk 6 above)
Comment (weird94) [reply packages/core/src/docs/data-model/text-x/__tests__/transform.spec.ts:749]: 长度为1

**Pull request closed without merging**